docs/BackupRDB.md

```diff
@@ -0,0 +1,14 @@
+# Backup RDB guide
```

Comment:
Following the guidance in [Google Developer Documentation Style Guide ("Heading and title text")](https://developers.google.com/style/headings#heading-and-title-text), how about we use the following?:

```suggestion
# Back up an RDB in a Kubernetes environment
```